packages/react-iv-viewer/src/FullScreenViewer.tsx

```diff
@@ -11,7 +11,7 @@ export const FullScreenViewer: React.FC<FullScreenViewerProps> = ({
   refreshOnResize = true,
   zoomOnMouseWheel = true,
   hasZoomButtons = true,
-  zoomStep,
+  zoomStep = 50,
   onLoad,
   alt = 'Zoomable Image',
   ...rest
```

Comment:
**Suggestion**
In this spread should be after cursor: pointer, so user can override if they wish.
```
<img
      {...rest}
      src={img}
      onClick={handleOnClick}
      style={{ ...rest.style, cursor: 'pointer' }}
      alt={alt}
    />
```